### PR TITLE
Browser session UI

### PR DESCRIPTION
**Pull request by kerryarchibald**
Fixes https://github.com/matrix-org/matrix-authentication-service/issues/1545

Use the same generic session element for browser sessions

Screenshot 2023-08-28 at 13 47 53

### CODE REVIEW

File: frontend/src/components/UserHome/UserHome.tsx [hunk 1]
```diff
@@ -72,48 +73,47 @@ const UserHome: React.FC<{
...
-    <>
+    <BlockList>
       {data.unverifiedEmails.totalCount > 0 && !dismiss && (
         <Alert type="critical" title="Unverified email" onClose={doDismiss}>
           You have {data.unverifiedEmails.totalCount} unverified email
           address(es). <Link route={{ type: "profile" }}>Check</Link>
         </Alert>
       )}
       {/* This is a short term solution, so I won't bother extracting these blocks into components */}
-      <section className={styles.sessionList}>
```
Comment (sandhose) [inline frontend/src/components/UserHome/UserHome.tsx:83]: That class is still in the CSS module

**Review by sandhose — CHANGES_REQUESTED** [review]
Just a leftover class in the CSS module, otherwise LGTM

**Comment by kerryarchibald** [comment]
Thanks @sandhose
I've enabled auto-merge

**Comment by kerryarchibald** [comment]
Oops, didn't realise it would merge without approval.

**Pull request merged**